src/index.js

```diff
@@ -1 +1 @@
-import './style.css';
+import './style.css';
```

Comment:
- [ ] Kindly [Check the project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md) and work accordingly

Reply:
sorry I requested to review to wrong day that is why i did not implemented API stuff